Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -229,13 +242,12 @@ - (UIActivityIndicatorViewStyle)activityIndicatorViewStyle {
 */
 - (void)startAnimating {
     RunSynchronouslyOnMainThread(^{
-        if (_isAnimating) {
-            return;
+        if (!_isAnimating) {
+            _isAnimating = YES;
+            [self setHidden:NO];
```

Comment:
>   [self setHidden:NO]; [](start = 9, length = 23)

Does startAnimating set the control back to visible if the app explicitly set isHidden:YES?